client/modules/User/pages/DashboardView.jsx

```diff
@@ -124,7 +125,7 @@ class DashboardView extends React.Component {
           <div className="dashboard-header__header">
             <h2 className="dashboard-header__header__title">{this.ownerName()}</h2>
             <div className="dashboard-header__nav">
-              <DashboardTabSwitcher currentTab={currentTab} isOwner={isOwner} username={username} />
+              <DashboardTabSwitcher currentTab={currentTab} isOwner={isOwner} username={username} translate={this.props.t} />
```

Comment:
See comment above, I think `DashboardTabSwitcher` should handle translations without needing the function to be passed in.